src/pallets/static/style.css

```diff
@@ -8,27 +13,105 @@ header nav a.logo {
 }
 
 header nav a.logo img {
-  height: 3em;
-  max-width: none;
+  width: 3em;
+}
+
+header nav {
```

Comment:
What is this changing? The screenshot looks the same, and pico already handles nav layout.

Reply:
I was moving control away from Pico here with the foresight that you would be changing the flex direction to column on mobile view.